module/zstd_zfs/include/limits.h

```diff
@@ -0,0 +1,7 @@
+/* This file is in the public domain */
+/* $FreeBSD$ */
```

Comment:
Do we really need these relatively useless `$FreeBSD$` markers? They don't expand in this source tree...

Reply:
These files were taken verbatim from FreeBSD. We can remove these lines if that makes sense.

Reply:
Personally, I'd prefer them gone, as they look silly unexpanded.